bcda/auth/alpha.go

```diff
@@ -179,7 +201,8 @@ func (p AlphaAuthPlugin) RequestAccessToken(creds Credentials, ttl int) (Token,
 		userUUID = user.UUID
 		acoUUID = user.ACOID
 	} else {
-		aco, err := getACOFromDB(creds.ClientID)
+		var aco models.ACO
```

Comment:
why declare it this way?  just curious.  we don't seem to use the empty aco afterwards

Reply:
The aco is used on the next line? It's catching the return value from the function, which is not the typical pattern of how we get objects out of the db. I can't use := on the next line because err is actually declared at the top of the function. I decided against moving the decl of aco up because it's only used inside this else block. I suppose another option would have been to just shadow the err in the outer block, but shadowed vars == bad juju.